index.bs

```diff
@@ -4821,30 +4828,35 @@ Methods</h4>
 		<div algorithm="AudioBufferSourceNode.start()">
 			<span class="synchronous">When this method is called, execute these steps:</span>
 
-			1. <span class="synchronous">If <code>stop</code> has been called on this node, or if an
-				earlier call to <code>start</code> has already occurred, an
+			1. <span class="synchronous">If this {{AudioBufferSourceNode}} internal
+				slot {{AudioScheduledSourceNode/[[source started]]}} is true, an
```

Comment:
`<code>true</code>`

FYI I believe the `<code>` tag be replaced with a back tick.